docs/users/SystemRequirements.md

```diff
@@ -0,0 +1,43 @@
+# System Requirements
+!!! note "This page does not include the requirements to run any supported games. Please refer to the game store page for details."
+
+This page lists the minimum and recommend system requirements to install and run the app on all supported platforms. 
+
+## Windows 
+
+|| Minimum | Recommended |
+| --- | --- | --- |
+| OS | Windows 10 or higher (64-bit OS required) | Windows 10/11 with updates |
+| Processor | Intel®️ Core™️ i5-7500 | Intel®️ Core™️ i7-7500 |
+| Memory | 4 GB RAM | 16 GB RAM |
+| Graphics | 2GB VRAM | 4GB VRAM |
+| Storage | 10GB available space* | 200GB available space* |
+
+<sub>\* Storage space varies by the number of games managed, the size of the managed games and the mods installed.</sub>
+
+## SteamOS + Linux 
+|| Minimum | Recommended |
+| --- | --- | --- |
+| OS | Ubuntu 12.04 LTS | A currently supported Linux distribution |
+| Processor | Intel®️ Core™️ i5-7500 | Intel®️ Core™️ i7-7500 |
+| Memory | 4 GB RAM | 16 GB RAM |
+| Graphics | 2GB VRAM | 4GB VRAM |
+| Storage | 10GB available space* | 200GB available space* |
```

Comment:
This is also just straight up wrong. Ubuntu 12.04 is ancient, and **not** supported. I don't know where you got these values from, but you should just use the original requirements I wrote down.

Reply:
I can update this page to just have vague recommended specs but the app will not run well on a potato so it's important to set expectations.

Reply:
Just don't put any specs in there. Instead, you can write down that the App is bottlenecked by disk speed, so an SSD or NVME is recommended. The App is also multithreaded, and will make use of all available cores.

Reply:
I have to agree here, saying we support Ubuntu 12 means we actually support it, and I seriously doubt we'll do that. I'm also not sure if we want to tie the minimum requirements to Ubuntu when Steam Deck runs arch. So we may just want to say "we support Linux" or something of that nature

Reply:
fixed